src/classic/types/__tests__/ReactPropTypes-test.js

```diff
@@ -371,7 +374,21 @@ describe('ReactPropTypes', function() {
         <MyComponent />
       ]);
 
-      // Object of rendereable things
+      // Object of renderable things
+      var frag = ReactFragment.create;
+      typeCheckPass(PropTypes.node, frag({
+        k0: 123,
+        k1: 'Some string',
+        k2: <div />,
+        k3: frag({
+          k30: <MyComponent />,
+          k31: frag({k310: <a />}),
+          k32: 'Another string'
+        })
+      }));
+      expect(console.warn.calls).toEqual([]);
+
+      // This should also pass, though it warns
       typeCheckPass(PropTypes.node, {
```

Comment:
It's a little weird to me how this warns, but this seems to match the behavior of key warnings on element creation where it warns if any of the object values is an element.